examples/connect/CMakeLists.txt

```diff
@@ -7,20 +7,12 @@
 
 # This is a very simple example of using SOCI in a CMake-based project.
 
-cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 project(SOCIExampleConnect)
 
-find_package(Soci REQUIRED)
+find_package(SOCI REQUIRED)
 
 add_executable(connect connect.cpp)
 
-# Note that depending on the SOCI library automatically pulls in the required
-# SOCI compilation options too, i.e. there is no need to explicitly use
-# target_include_directories().
```

Comment:
I would keep this comment, why remove it? It can be useful to people new to CMake.